src/main/java/sparksoniq/jsoniq/runtime/iterator/functions/DynamicFunctionCallIterator.java

```diff
@@ -40,15 +45,23 @@
     private List<RuntimeIterator> _fnArguments;
```

Comment:
@ghislainfourny I think we should stick to regular Java names with full words -> functionArguments. What do you think?

Reply:
Yes, I agree.

Reply:
Nice catch! I will address this and double check the last few PRs as well.